_episodes/04-wildcards-pipes.md

```diff
@@ -196,7 +204,7 @@ Awesome, let's try that with a more complicated command, like `wc -l`.
 $ wc -l * > word_counts.txt
 $ cat word_counts.txt
 ```
-{: .language-bash}
+{: .bash}
```

Comment:
Why are these being changed? It looks like the originals are correct according to https://carpentries.github.io/lesson-example/04-formatting/index.html

Reply:
Vestigial from a time when `lesson_check.py` failed on `language-bash`. Thanks for catching & correcting this.

Reply:
Fixed by 83a8b56